snippets/base/templates/base/snippets_custom_name_with_tags.jinja

```diff
@@ -0,0 +1,15 @@
+<div id="name">
+  <a href="{{ url('admin:base_asrsnippet_change', object_id=obj.id) }}">
+    {{ obj.name }}
+  </a>
+</div>
+<ul class="tags">
+  {% for tag in obj.tags.all().order_by('name') %}
+    <li>
+      <a href="{{ url('admin:base_asrsnippet_changelist') }}?tag={{ tag.name }}" class="tag">
```

Comment:
note to self: make so it respects existing filtering

Reply:
Not important atm and we get the same functionality the the list filter on the side 
![image](https://user-images.githubusercontent.com/584352/58941026-9dd5a280-8783-11e9-84a1-04f73fab1404.png)